XRTK-Core/Packages/com.xrtk.core/package.json

```diff
@@ -10,7 +10,7 @@
     "Mixed Reality",
     "DI"
   ],
-  "version": "0.1.32",
+  "version": "0.2.0",
```

Comment:
Not sure if this was discussed, but I think this should still be in the 0.1 range until we are ready for the minor release upgrade

Reply:
We are ready. I always bump the version number like this for breaking change PRs